docs/patterns/kb-patterns.md

```diff
@@ -0,0 +1,13 @@
+You can find some useful knowledge base constructions patterns in `kb/pattern`.
+There are examples to formalize classes, relations, instances, answer rules, classification rules and structures to answer to new messages.
+
+In the `gwf` subdirectory you can find examples in scg. You can load `gwf` files to the scg-editor, synchronize and get new answers online.
+
+For example, here you need to replace identifiers `concept_your_message_class_name`, `concept_your_class_identifier`, `concept_phrase_about_your_theme`,
+and `lr_message_about_your_class` to your actual nodes. Also, you can change any relations if you need.
+<img src="../images/answer_rule_example.png"></img>
+
+In the `scs` subdirectory you can find the same examples in scs.
+
+In the `science` subdirectory you can find KB fragments in scg that serve as an example ontology. If you'll build these files (add this directory to `repo.path`), you'll get answers to the questions related to science.
+Use it as a concrete example.
```

Comment:
I didn't quite get what did you mean by this. I have a guess that you've meant "use it as a specific example", but a specific example in some kind of system, report or other activity, right? Looks like some context is necessary. Also word "concrete" has big ambivalence in English, so it's better to replace it with "specific".